keps/sig-apps/85-Graduate-PDB-to-Stable/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: KEP Template
+kep-number: 85
+authors:
+  - "@bsalamat"
+  - "@mortent"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-scheduling
+status: implementable
+creation-date: 2019-03-18
+reviewers:
+  - "@liggitt"
+  - "@kow3ns"
+  - "@soltysh"
+  - "@janetkuo"
+  - "@wojtek-t"
+approvers:
+  - "@kow3ns"
+  - "@liggitt"
+  - "@wojtek-t"
+prr-approvers:
+  - "@wojtek-t"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
```

Comment:
The target stage for this KEP is `stable`, I think?

Reply:
Yeah, I agree. I have updated the KEP.